manifest.json

```diff
@@ -1,6 +1,6 @@
 {
    "name": "LinkBot",
-   "version": "1.0.6",
+   "version": "1.0.7",

```

Comment:
Could you also update the version in package.json? That version doesn't really do anything and it's been out of date (oops) but might as well keep the two in sync.